portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,10 +34,55 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private String name;
+  ArrayList<String> comments = new ArrayList<String>();
+  List<Task> tasks = new ArrayList<>();
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String name = "Bless";
+    String json = convertToJson();
+    response.setContentType("text/html;");
+    response.getWriter().println(json);
```

Comment:
You're setting the response before querying the comments from the database. That's why it's not showing them on the webpage. Make sure to only do that at the very end of `doGet`, once you've populated `tasks`.

Ideally, it would be nice to pass `tasks` as an argument to `convertToJson` instead of using a field variable.